clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -36,15 +36,19 @@ describe("System management with Plus features", () => {
       cy.getSelectValueContainer("input-vendor_id");
     });
 
-    it("contains dictionary entries", () => {
-      cy.selectOption("input-vendor_id", "Aniview LTD");
+    it("contains type ahead dictionary entries", () => {
```

Comment:
turns out these were legitimately failing tests! looks like they weren't updated after the typeahead work